CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Reduced the complex tolerance in the mode solver below which permittivity is considered lossless, in order to correctly compute very low-loss modes.
 -`HeatChargeSimulation` supersedes `HeatSimulation`. Though both of the can be used for Heat simulation interchangeably, the latter has been deprecated and will disappear in the future. 
 - `FluidSpec` and `SolidSpec` are now deprecated in favor of `FluidMedium` and `SolidMedium`. Both can still be used interchangeably.
+- Exclude `FluxMonitor` frequencies from adjoint monitors since we do not differentiate through `FluxData`
```

Comment:
"adjoint monitors" -> "adjoint design region gradient monitors". just to be really specific here that we mean the design region fields not the fields used to make the adjoint sources.